video/vdu_buffered.h

```diff
@@ -1060,4 +1072,203 @@ void VDUStreamProcessor::bufferCopyAndConsolidate(uint16_t bufferId, std::vector
 	debug_log("bufferCopyAndConsolidate: copied %d bytes into buffer %d\n\r", length, bufferId);
 }
 
+// Write compressed output to a temporary buffer
+//
+static void local_write_compressed_byte(void* p_cd, uint8_t comp_byte) {
```

Comment:
I'm wondering if this, and `local_write_decompressed_byte` wouldn't better belong inside `compression.h` ?

Reply:
Agreed. It doesn't specifically involve anything that is "buffer" related, just compression related.